tests/integration/standard/test_authentication.py

```diff
@@ -42,12 +43,19 @@ def setup_module():
         log.debug("Starting ccm test cluster with %s", config_options)
         start_cluster_wait_for_up(ccm_cluster)
 
+    # PYTHON-1328
+    #
+    # Give the cluster enough time to startup (and perform necessary initialization)
+    # before executing the test.
+    if CASSANDRA_VERSION > Version('4.0-a'):
+        time.sleep(10)
```

Comment:
This wait was happening within a test method but this relies on tests being executed in a specific order which is... not great.  Moving to module initialization logic should resolve that issue.